src/misc.c

```diff
@@ -782,3 +782,72 @@ static int luv_clock_gettime(lua_State* L) {
   return 1;
 }
 #endif
+
+#if LUV_UV_VERSION_GEQ(1, 49, 0)
```

Comment:
This is suspicious, Libuv documentation lists these as being introduced in 1.47.0

Reply:
They were added to `uv.h` in 1.47.0, but were not exported until 1.49.0: https://github.com/libuv/libuv/pull/4437

Reply:
These API introduced in 1.47.0, but document and public in uv.h v1.49.0, Will compile fail if we set 1.47.0.